src/vsr/replica.zig

```diff
@@ -1034,6 +1039,41 @@ pub fn ReplicaType(
             const replica_index = options.replica_index;
             assert(replica_index < node_count);
 
+            self.replica_count = replica_count;
+            self.standby_count = standby_count;
+            self.node_count = node_count;
+            self.replica = replica_index;
+
+            // Ensure each replica can send have a maximum of two inflight WAL repair messages
```

Comment:
I think this one exceeds the budget of verbs in a phrase, gotta pick only one of "send" or "have"!